hunting-buddy.lic

```diff
@@ -76,6 +78,16 @@ class HuntingBuddy
     bput("stow my #{description}", 'You put')
   end
 
+  def check_prebuffs
+    return unless @settings@settings.pre_hunt_buff 
+    return unless !@settings.waggle_sets.nil?
```

Comment:
This removes the need for the boolean, if you have a prebuff set, it will auto buff you, where you are standing or at the room specified if there is one. 

`   return unless @settings.waggle_sets['prebuff']`